oauth/client.go

```diff
@@ -0,0 +1,84 @@
+// Package oauth provides the OAuth APIs
+package oauth
+
+import (
+	"fmt"
+	stripe "github.com/stripe/stripe-go"
+	"github.com/stripe/stripe-go/form"
+	"net/http"
+)
+
+// Client is used to invoke /oauth and related APIs.
+type Client struct {
+	B   stripe.Backend
+	Key string
+}
+
+// AuthorizeURL builds the OAuth authorize URL.
+func AuthorizeURL(params *stripe.AuthorizeURLParams) string {
```

Comment:
Will `AuthorizeURL` ever be needed by a user integration?

If not, my vote is that we keep it unexported (i.e. rename to `authorizeURL`). The tests will still be able to get at it because they're in the same package.

Reply:
Yeah this is a tool for users to create their AuthorizeURL that they will redirect to.

Reply:
Ah yes, now that I think about how OAuth works, that makes sense :)